packages/frontend/src/components/Explorer/CustomMetricModal/utils/index.ts

```diff
@@ -95,6 +97,11 @@ export const prepareCustomMetricData = ({
             ? { round: dimension.round }
             : defaultRound;
 
+    const percentile =
+        shouldCopyFormatting && dimension.percentile
+            ? dimension.percentile
+            : metricPercentile;
```

Comment:
wouldn't this always send `metricPercentile` `50` if it was non-percentile?

Reply:
Hm, yea. It should be benign, but good catch. Will change.